.github/workflows/pr-workflow.yaml

```diff
@@ -0,0 +1,10 @@
+name: CI for PRs
+
+on: pull_request
```

Comment:
Before this PR is merged, we'll want to switch this trigger to run on `pull-request/**` branches. Similar to https://github.com/rapidsai/node/blob/39b4c72005a0c40160125249f36231c63e83aa7f/.github/workflows/main.pr.yml#L3-L6